lms/templates/dashboard/_dashboard_credit_info.html

```diff
@@ -77,11 +74,11 @@
         </p>
         <div class="credit-action">
             % if credit_btn_label:
-                <a class="btn credit-btn ${credit_btn_class}" href="${credit_btn_href}" target="_blank" data-course-key="${credit_status['course_key']}" data-user="${user.username}" data-provider="${credit_status['provider_id']}">
-                    ${credit_btn_label}
+                <a class="btn credit-btn ${credit_btn_class | h}" href="${credit_btn_href | h}" target="_blank" data-course-key="${credit_status['course_key'] | h}" data-user="${user.username | h}" data-provider="${credit_status['provider_id'] | h}">
+                    ${credit_btn_label | h}
                 </a>
             % endif
-            <div class="message-copy credit-msg ${credit_msg_class}">${credit_msg}</div>
+            <div class="message-copy credit-msg ${credit_msg_class | h}">${credit_msg}</div>
```

Comment:
`credit_msg` should not be escaped as it includes links to the provider's website. We control all of the data—URL and provider name—for this message, so it is safe.